docs/versioned/integrations/aws.md

```diff
@@ -188,21 +180,22 @@ However, if you are streaming data to Iceberg, this will easily create a lot of
 Therefore, it is recommended to turn off the archive feature in Glue by setting `glue.skip-archive` to `true`.
 For more details, please read [Glue Quotas](https://docs.aws.amazon.com/general/latest/gr/glue.html) and the [UpdateTable API](https://docs.aws.amazon.com/glue/latest/webapi/API_UpdateTable.html).
 
-#### DynamoDB for Commit Locking
+#### Optimistic Locking
 
-Glue does not have a strong guarantee over concurrent updates to a table. 
-Although it throws `ConcurrentModificationException` when detecting two processes updating a table at the same time,
-there is no guarantee that one update would not clobber the other update.
-Therefore, [DynamoDB](https://aws.amazon.com/dynamodb) can be used for Glue, so that for every commit, 
-`GlueCatalog` first obtains a lock using a helper DynamoDB table and then try to safely modify the Glue table.
+By default, Iceberg uses Glue's optimistic locking for concurrent updates to a table.
+With optimistic locking, each table has a version id. 
+If you retrieve the table metadata, Iceberg records the version id of that table. 
```

Comment:
should not say "you", prefer to use third-person like "users"

Reply:
I have made the changes.